Makefile

```diff
@@ -34,6 +34,7 @@ $(call build-image,ocp-cluster-kube-scheduler-operator,$(IMAGE_REGISTRY)/ocp/4.2
 $(call add-bindata,v4.1.0,./bindata/v4.1.0/...,bindata,v410_00_assets,pkg/operator/v410_00_assets/bindata.go)
 
 e2e: GO_TEST_PACKAGES :=./test/e2e
+e2e: GO_TEST_FLAGS += -v
```

Comment:
I don't think we need that verbosity, if the test fails, we'll get it, otherwise it's not needed.

Reply:
removed it